src/xenia/hid/winkey/winkey_input_driver.cc

```diff
@@ -25,10 +25,29 @@
 #include "winkey_binding_table.inc"
 #undef XE_HID_WINKEY_BINDING
 
+DEFINE_int32(keyboard_passthru_user_index, -1,
+             "Allows keyboard to be assigned as virtual keyboard to user with "
+             "specific index. This also forces keyboard to be assigned to that "
+             "slot to be interpreted as controller. Possible values: -1 - "
+             "Disabled (Keyboard is in "
+             "gamepad mode), [0, 3] - Keyboard is assigned as VK for that user",
+             "HID");
+
 namespace xe {
 namespace hid {
 namespace winkey {
 
+bool __inline IsPassThruForUserEnabled(uint32_t user_index) {
```

Comment:
can't this just be user_index == cvars::keyboard_passthru_user_index
is -1 possible? it doesnt seem like it should be

Reply:
Yeah it shouldn't. -1 is used but as 0xFF to define any user